200.blacklists/metaports.yaml

```diff
@@ -132,6 +132,8 @@
 - { remove: true,   name: xorg-truetype-fonts }
 - { remove: true,   name: xorg-x11-apps }
 - { remove: true,   name: xorg-x11-utils }
+- { remove: true,   name: xserver-xorg }
```

Comment:
These may remove real packages in other repos, so let's limit these two with `ruleset: chimera`.